site/_layouts/blog-page.html

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
just curious about this file: if it’s not provided by the user, is there a “default” blog page template?

Reply:
![image](https://user-images.githubusercontent.com/6325220/124081886-56966a80-da4c-11eb-85fa-d3d827b6f882.png)

vs

![image](https://user-images.githubusercontent.com/6325220/124081973-7594fc80-da4c-11eb-8a8a-60400120475e.png)